tortoise/contrib/postgres/json_functions.py

```diff
@@ -14,37 +44,106 @@ def postgres_json_contained_by(field: Term, value: str) -> Criterion:
     return BasicCriterion(JSONOperators.CONTAINED_BY, field, ValueWrapper(value))
 
 
-operator_keywords = {
+operator_keywords: dict[str, Callable[..., Criterion]] = {
     "not": not_equal,
     "isnull": is_null,
     "not_isnull": not_null,
+    "in": is_in,
+    "not_in": not_in,
+    "gte": cast(Callable[..., Criterion], operator.ge),
+    "gt": cast(Callable[..., Criterion], operator.gt),
+    "lte": cast(Callable[..., Criterion], operator.le),
+    "lt": cast(Callable[..., Criterion], operator.lt),
+    "range": between_and,
+    "contains": contains,
+    "startswith": starts_with,
+    "endswith": ends_with,
+    "iexact": insensitive_exact,
+    "icontains": insensitive_contains,
+    "istartswith": insensitive_starts_with,
+    "iendswith": insensitive_ends_with,
+    "year": extract_year_equal,
+    "quarter": extract_quarter_equal,
+    "month": extract_month_equal,
+    "week": extract_week_equal,
+    "day": extract_day_equal,
+    "hour": extract_hour_equal,
+    "minute": extract_minute_equal,
+    "second": extract_second_equal,
+    "microsecond": extract_microsecond_equal,
```

Comment:
Don't we need more test for all these new operators?

Reply:
@abondar there are tests in `tests/contrib/postgres/test_json.py` for all of these. Can you suggest any particular test cases you want me to add?

Reply:
Sorry, I am in wrong here, for some reason I only saw file with generic test_json, and didn't notice file with postgres-specific test_json